docs/DeployingOnCentOS7.md

```diff
@@ -0,0 +1,291 @@
+
+## Install the packages
+
+If you followed the guide correctly so far, you should have the Mesos rpm packages on each of your master and slave hosts.
+
+Before installing these packages, you must have the Extra Packages for Enterprise Linux (or EPEL) packages installed.
+
+    $ sudo yum -y install epel-release
+
+Now your server should be ready to install the Mesos packages.
+
+    $ sudo yum -y install *.rpm
+
+# Run the Backend Services
+
+You will need to run one instance of both, etcd and ZooKeeper. You can run etcd and ZooKeeper on any host you choose, but we recommend running them on your master.
+
+Both services will be run in Docker containers.
+
+Also, keep note of the IP address of the host you deploy on because we will need this later when configuring our Mesos masters and slaves.
+
+## ZooKeeper
+
+    $ sudo docker pull jplock/zookeeper:3.4.5
+    $ sudo docker run --detach --name zookeeper -p 2181:2181 jplock/zookeeper:3.4.5
+
+## etcd
+
+`etcd` needs your fully qualified domain name to start correctly.
+
+    $ sudo docker pull quay.io/coreos/etcd:v2.2.0
+    $ export FQDN=`hostname -f`
+    $ sudo mkdir -p /var/etcd
+    $ chcon -Rt svirt_sandbox_file_t /var/etcd
+    $ sudo docker run --detach --name etcd --net host -v /var/etcd:/data quay.io/coreos/etcd:v2.2.0 \
+     --advertise-client-urls "http://${FQDN}:2379,http://${FQDN}:4001" \
+     --listen-client-urls "http://0.0.0.0:2379,http://0.0.0.0:4001" \
+     --data-dir /data
+
+# Install and Configure Calico on the Slaves
+
+Calico will need to be running on each of your slaves in order to perform Calico networking. 
+
+To obtain the Calico files, you will need `wget` installed. If you haven't already done so, download the tool with `yum -y install wget`.
+
+## Download the Calico Isolator
+
+    $ wget https://github.com/projectcalico/calico-mesos/releases/download/v0.1.1/calico_mesos
+    $ chmod +x calico-mesos
+    $ sudo mkdir /calico
+    $ sudo mv calico_mesos /calico/calico_isolator
+
+## Create the modules.json Configuration File
+
+    $ cat > modules.json <<EOF
+    {
+      "libraries": [
+        {
+          "file": "/opt/net-modules/libmesos_network_isolator.so",
+          "modules": [
+            {
+              "name": "com_mesosphere_mesos_NetworkIsolator",
+              "parameters": [
+                {
+                  "key": "isolator_command",
+                  "value": "/calico/calico_isolator"
+                },
+                {
+                  "key": "ipam_command",
+                  "value": "/calico/calico_isolator"
+                }
+              ]
+            },
+            {
+              "name": "com_mesosphere_mesos_NetworkHook"
+            }
+          ]
+        }
+      ]
+    }
+    EOF
+    $ sudo mv modules.json /calico/
+
+## Run Calico Node
+
+Download `calicoctl`, Calico's command line tool. We will use `calicoctl` to spin up calico-node in a docker container on each slave. 
```

Comment:
Most people aren't going to know what "spinning up a calico-node in a docker container" means. Reword this to give a glimpse of background for calico-node (Core calico-processes packaged up in a docker container).